jsLearn/arrayMethods.js

```diff
@@ -51,21 +51,30 @@ console.log(copySorted(arr5), arr5)
 
 
 // 6. Create an extendable calculator
-        // xndri erkrord mas@ mnuma lucelu
 
 function Calculator() {
+    this.operators = {
+        "+" : (a, b) => a + b,
+        "-" : (a, b) => a - b,
+    }
 
     this.calculate = function(str) {
-        return eval(str)
+        let stringItems = str.split(" ")
+
+        return this.operators[stringItems[1]](+stringItems[0], +stringItems[2])
```

Comment:
that's okay, but can you use destructuring for more readable code?

Reply:
this way?

Reply:
no, destructuring, not just assignment

Reply:
done